.github/workflows/lint.yml

```diff
@@ -29,13 +29,13 @@ jobs:
         run: composer run lint
 
   php-cs-fixer:
-    name: php-cs
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         php-versions: ['7.4']
```

Comment:
btw php version is in `'` and node version is not. Is there a difference or can we unify?

Reply:
No idea :woman_shrugging: I guess yaml is smart enough to see it as string?

Reply:
Well smart is probably not the right word here https://medium.com/@lefloh/lessons-learned-about-yaml-and-norway-13ba26df680 :see_no_evil:

Reply:
But it should not matter in that case if it is quoted or not.